website/docs/getting-started/insecure-registry.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 10
+title: Adding an insecure registry
+description: Modifying Podman for an insecure registry
+keywords: [podman desktop, podman, containers, registry, podman machine]
+tags: [pods]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Adding an Insecure Registry to Podman Desktop
+
+In this guide, you will learn how to add an insecure registry to Podman Desktop. An insecure registry allows you to pull and push container images over an unencrypted HTTP connection instead of a secure HTTPS connection. Please note that using an insecure registry can expose your data to security risks, so it's recommended to use secure connections whenever possible.
+
```

Comment:
I think insecure is not only unencrypted HTTP connection but also just a self-signed certificate (so we don't trust the certificate)